app/controllers/api/migration_analytics_manifest_controller.rb

```diff
@@ -0,0 +1,34 @@
+def parse_json_if_valid(json)
+    return JSON.parse(json)
+  rescue JSON::ParserError => e
+    return :invalid
+end
+
+module Api
+  class MigrationAnalyticsManifestController < BaseController
+    def index
+      tmp_manifest_path = '/tmp/migration-analytics-manifest.json'
+      tmp_manifest = File.file?(tmp_manifest_path) ? parse_json_if_valid(File.read(tmp_manifest_path)) : nil
+      tmp_manifest_valid = tmp_manifest != nil && tmp_manifest != :invalid
```

Comment:
Maybe we take this one step at a time and just return the builtin manifest for now, then expand to handle user uploaded also

Reply:
I can break this into two PRs if you want. But I know Miguel was asking for us to make sure there is always a way for the user to manually upgrade the included manifest, in case we ship this feature and then release some change to the cloud.redhat.com part of Migration Analytics that depends on a newer manifest that hasn't been released built-in yet.

Maybe he'll have an opinion here.. cc @mperezco

Reply:
I don't like having it in `/tmp` because some systems clear that on reboot and on a schedule as a cron task, but I don't want to hold this PR up while we discuss where the user could upload it.

Reply:
I don't like having it in /tmp because some systems clear that on reboot and on a schedule as a cron task, but I don't want to hold this PR up while we discuss where the user could upload it.

Reply:
Removed.